packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -290,6 +291,11 @@ class Tooltip extends StatefulWidget {
   /// for mouse pointer exits the widget.
   final Duration? showDuration;
 
+  /// The length of the delay time before dismissing tooltips by tap.
+  ///
+  /// Defaults to 0 milliseconds (tooltips are dismissed immediately).
+  final Duration? tapToDismissDelayDuration;
```

Comment:
This feels like a weird api, is the goal to be able to have other gesture recognizer on the tool tip to fire? what if the gesture take longer than the duration to fire?  for example if someone add a long press gesture recognizer in tooltip.

Should we just have a flag to turn off tap dismissal?

Reply:
make sense, I changed it to a flag for tap dismissal